src/components/form.js

```diff
@@ -4,8 +4,9 @@ import { Formik } from "formik"
 import { Form, Datepicker, SubmitBtn, Input, Select } from "react-formik-ui"
 import * as Yup from "yup"
 import moment from "moment"
+import styled from "styled-components"
 
-import "./form.css"
+const StyledForm = styled.div``
```

Comment:
I think we don't need that component :)

Reply:
Well we had empty file `form.css` so I thought we will be doing some styles for it, but ya we can delete it ;)